packages/payload/src/fields/config/schema.ts

```diff
@@ -366,6 +366,9 @@ export const relationship = baseField.keys({
       Label: componentSchema,
     }),
     isSortable: joi.boolean().default(false),
+    sortOptions: joi
+      .alternatives()
+      .try(joi.string(), joi.object().pattern(joi.string(), joi.string())),
```

Comment:
Can we make this better? Really sortOptions should only be allowed to be an object if relationTo is an array.